plugins/opener/src/reveal_item_in_dir.rs

```diff
@@ -54,56 +96,79 @@ mod imp {
         },
     };
 
-    pub fn reveal_item_in_dir(path: &Path) -> crate::Result<()> {
-        let file = dunce::simplified(path);
+    pub fn reveal_items_in_dir(paths: &[PathBuf]) -> crate::Result<()> {
+        if paths.is_empty() {
+            return Ok(());
+        }
+
+        let first_path = dunce::simplified(&paths[0]);
```

Comment:
I think we could remove all the `dunce::simplified` calls and use `dunce::canonicalize` instead of `PathBuf` on the ones above